test/browser_tests/cucumber-new/features/suites/apps/financial-well-being-survey.feature

```diff
@@ -0,0 +1,112 @@
+Feature: Verify financial well-being survey page works according to requirements
+  As a first time visitor to the financial well-being survey page
+  I want to Answer ten questions to measure my current financial well-being
+  So that I can see steps for me to take to improve it
+
+@s1 @age18-61 @Never
+Scenario: Financial Well-being survey 1 for age range 18-61
+  Given I visit the consumerfinance consumer-tools/financial-well-being URL
+  When I Click on "Completely" for question "I could handle a major unexpected expense"
+  And I Click on "Completely" for question "I am securing my financial future"
+  And I Click on "Not at all" for question "Because of my money situation, I feel like I will never have the things I want in life"
+  And I Click on "Completely" for question "I can enjoy life because of the way I’m managing my money"
+  And I Click on "Not at all" for question "I am just getting by financially"
+  And I Click on "Not at all" for question "I am concerned that the money I have or will save won’t last"
+  And I Click on "Never" for question "Giving a gift for a wedding, birthday or other occasion would put a strain on my finances for the month"
+  And I Click on "Always" for question "I have money left over at the end of the month"
+  And I Click on "Never" for question "I am behind with my finances"
+  And I Click on "Never" for question "My finances control my life"
+  And I Select my age group of "18-61"
+  And I Select how I completed the questionnaire as "I read and answered the questions myself"
+  And I click on the "Get my score" button on the financial well-being quiz
+  Then I should find the text "financial well-being" on the page
```

Comment:
I think for this file, it would be better for the assertion to test the actual score that is returned when these answers are chosen. It seems like having six different scenarios all testing the same assertion is a little redundant, and the key bit of functionality for this tool here is computing the score.

Reply:
Agreed. I will update this feature file such that the assertion will test the score that is returned. By making this change I will verify if the score changes based on age group and how the questions were answered.